student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(age, classroom, name, parent_permission)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def classroom(classroom)
+    @classroom = classroom
+    classroom.students << self
+  end
```

Comment:
- [OPTIONAL] When setting the classroom for a student it also adds it to the classroom's students, Let's add the getter methods with attr_reader to allow us to access the ` classroom` from outside the scope of the class definition. Kindly add this line 👇🏻 
`attr_reader :classroom` at the top of your  Studen class

- You can also consider adding a condition while pushing 👇🏻 

```ruby
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```